src/PrintConfig.cc

```diff
@@ -75,19 +76,36 @@ bool PrintConfig::PreserveIncludes() const
 /////////////////////////////////////////////////
 bool PrintConfig::SetRotationSnapToDegrees(unsigned int _interval,
                                            double _tolerance)
+{
+  sdf::Errors errors;
+  bool result = this->SetRotationSnapToDegrees(_interval,
+                                               _tolerance,
+                                               errors);
+  throwOrPrintErrors(errors);
+  return result;
+}
+
+/////////////////////////////////////////////////
+bool PrintConfig::SetRotationSnapToDegrees(unsigned int _interval,
+                                           double _tolerance,
+                                           sdf::Errors &_errors)
 {
   if (_interval == 0 || _interval > 360)
   {
-    sdferr << "Interval value to snap to must be larger than 0, and less than "
-           << "or equal to 360.\n";
+    std::stringstream ss;
+    ss << "Interval value to snap to must be larger than 0, and less than "
+       << "or equal to 360.\n";
```

Comment:
nit: I think we can drop the newline at the end.  Same with the next error message below.

Reply:
Removed.